### ISSUE TITLE
Wrong path for "env" in SystemConfigurator's Grub.pm

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#255]
**Reported by bli on 29 Aug 2006 05:52 UTC**
Getting the following error on the virtual console during testing on SUSE Linux 10.0 with branch-5.0 r5320:

```
Can't exec "/bin/env": No such file or directory at /usr/lib/systemconfig/Boot/Grub.pm line 120.
Could not run grub-install at /usr/lib/systemconfig/Boot.pm line 63
Killing off running processes.
```

Changing the path to `/usr/bin/env` seems to have solved the problem.

**Comment by dikim33** [comment]
**Comment by efocht on 29 Aug 2006 16:08 UTC**
The redhat-alike distros have env in /bin, some seem to have symlinks to /usr/bin, too (RHEL4). Not sure about Debian.

Leaving the command without the path to env leads to the failure of the tests during build, that's bad. So the cleanest option would be to detect the location with "which". I'll see whether I can extend that during the next few days.

**Comment by dikim33** [comment]
**Comment by efocht on 15 Sep 2006 13:13 UTC**
(In [5341]) Merged commit r5340 by efocht from 2006-09-15 12:29:45 +0200 (Fri, 15 Sep 2006)
with svnmerge_1by1 version v1.1 31.7.2006
from repository https://svn.oscar.openclustergroup.org/svn/oscar/trunk
to repository https://svn.oscar.openclustergroup.org/svn/oscar/branches/branch-5-0
Original commit message was:

Updating systemconfigurator to version 2.2.6-12ef.
This should fix bug #255.
SIS opkg version update to 3.7.3-10.

Closes #255
